src/vivarium_public_health/population/base_population.py

```diff
@@ -207,22 +207,43 @@ def _load_population_structure(self, builder: Builder) -> pd.DataFrame:
 
 
 class ScaledPopulation(BasePopulation):
-    """Component for scaling the population structure based on a scaling factor."""
+    """Component for scaling the population structure based on a scaling factor.
 
-    def __init__(self, scaling_factor: str | float | pd.DataFrame):
+    This component is to be used in place of the `BasePopulation` component when
+    all simulants in a simulation are a subset of a population and the user needs
+    to scale the population structure to account for the subset of the overall
+    population structure being treated as the total population within the simulation.
+    The scaling factor can be a dataframe passed in with python code or a string that
+    corresponds to an artifact key. If providing an artifact key, users can specify that
+    in the configuration file. For example:
+
+    .. code-block:: yaml
+
+    components:
+        vivarium_public_health:
+            population:
+                - ScaledPopulation("some.artifact.key")
+
+
+    """
+
+    def __init__(self, scaling_factor: str | pd.DataFrame):
         super().__init__()
         self.scaling_factor = scaling_factor
```

Comment:
Provide a docstring for the attr (triple-quoted and below the assignment). Mention the scaling factor is multiplicative